modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -272,8 +272,8 @@ public real_t Determinant()
         /// The returned vector contains the rotation angles in
         /// the format (X angle, Y angle, Z angle).
         ///
-        /// Consider using the <see cref="Quaternion()"/> method instead, which
-        /// returns a <see cref="Godot.Quaternion"/> quaternion instead of Euler angles.
+        /// Consider using the <see cref="GetRotationQuaternion"/> method instead, which
```

Comment:
This was changed because the GDScript documentation references the `GetRotationQuaternion()` method instead of the `GetQuaternion()`, this makes sense in GDScript since the `GetQuaternion()` method is not exposed. I think it makes sense in C# to direct users to the `GetRotationQuaternion()` method.

Reply:
Considering how it's used, I'm guessing that `GetQuaternion` only works when the Basis is a pure rotation matrix, so it's the same thing but with less safety and likely a bit faster. Plus it isn't exposed to GDScript. Do we want to keep this method exposed, or should it be changed to internal to avoid confusing users? @neikeq

Reply:
Actually, whether or not it's exposed, we should at the very least document the limitations of `GetQuaternion` in the XML doc comment.

Reply:
I think it's safe to make it private for 4.0 (or internal if it's used somewhere else).

Reply:
It's used in the Quaternion ctor that takes a Basis parameter so I'll make it internal.